### ISSUE TITLE
VibranceGUI not working anymore.

### ISSUE THREAD

**Issue by swekgg** [juv/vibranceGUI#61]
I have always used vibranceGUI without any issues but it stopped working for me. Changing the slider doesn't seem to do anything anymore, but the strange thing is that when I close the program, it does change the digital vibrance to the % it was set on while closing.

Example: I put 70% in vibranceGUI, nothing happens, I close vibranceGUI and the digital vibrance changes to 70%.

Any ideas?

Windows 10 version 1709 + GTX 1080 latest nvidia driver.

**Comment by All-github-usernames-taken** [comment]
- Same issue here on Windows 7 latest updates + latest nvidia driver (clean installed with DDU > latest version installed > rebooted twice)

- Slider settings save

- I then close app, reopen, settings still there

- but has no effect ingame

- tried redownloading it

- also tried deleting its folder from %appdata%

- in applicationdata.xml in its folder, 100% slider vibrance gives value of only 63 in that file. tried changing to 100 , no effect

**Comment by juvlarN** [comment]
@All-github-usernames-taken 
63 = maximum nvidia value, the nvidia driver only accepts values from 0-63 for the digital vibrance effect

which application are you trying? add another one, double click the icon, set your ingame vibrance value and then check if it works. Try for example your browser (chrome/firefox/whatever). If that works, try the application where is does NOT work in windowed/fullscreen windowed mode.

**Comment by swekgg** [comment]
Vibrance only changes when I close VibranceGUI. Other then that nothing works.

**Comment by juvlarN** [comment]
@swekgg which values are you using for "Windows vibrance value" and the "Ingame vibrance value"?

**Comment by swekgg** [comment]
Windows = default. Changing it in the control panel works without any issues. I have always used 100% on CSGO and Overwatch. The only thing I can think of what's causing this issue, is the fact that i formatted last week, a fresh Windows 10 install. Other then that I have no clue. I'm now using a different tool called "Saturation Toggler" which gets the job done but has limited features.

**Comment by All-github-usernames-taken** [comment]
@juvlarN

Apologies, I noticed that Nvidia Control panel has stopped working and no longer shows up when I right click on the desktop. As expected, when clean installing the latest Nvidia drivers after erasing the current ones with DDU in safe mode resolved the issue

**Comment by All-github-usernames-taken** [comment]
There was no issue with vibranceGUI in the first place, the Nvidia drivers had become corrupted or something but a clean install fixed it

**Comment by swekgg** [comment]
Maybe you can create you're own topic? Mine hasn't been fixed yet, DDU and fresh driver install does not change anything.

**Comment by All-github-usernames-taken** [comment]
It's pretty obvious we are experiencing different issues

**Comment by swekgg** [comment]
@juvlarN So the old version still seems to work... Any ideas?

**Comment by juvlarN** [comment]
@swekgg which vibranceGUI version are you using? Do you have a multi-monitor setup?

**Comment by juv** [comment]
Closing this, seems to be fixed. Reopen if necessary. Remember to update vibranceGUI if you are running an old version.

**Comment by Phenomz** [comment]
> Vibrance only changes when I close VibranceGUI. Other then that nothing works.

Hello, came across this thread from a Google search, I'm facing this exact same issue. Applications and games do work well, but the Windows vibrance only changes when VibranceGUI is closed, whereas it used to change instantly when the program was minimized. This started to happen with the latest NVIDIA driver update, it had happened to me before but I managed to solve it or it solved by itself, but this time nothing I've tried has worked so far. I've also updated drivers previously without this issue, so I don't really know what's the reason behind it
